srcpkgs/empathy/template

```diff
@@ -24,8 +25,19 @@ license="GPL-2.0-or-later, LGPL-2.1-or-later"
 homepage="http://live.gnome.org/Empathy"
 distfiles="${GNOME_SITE}/$pkgname/${version%.*}/$pkgname-$version.tar.xz"
 checksum=7d86942ce97edd10ade0e6ae6a210d35e4d627fe4d223377d71fd1840bc6e3a3
+patch_args=-Np1
 disable_parallel_build=yes
 
+do_check() {
+	# Disable checks, the only work with an irc server in place.
+	:
+}
```

Comment:
You can put `make_check=no` in the template directly.

Reply:
fixed.